modules/apps/item-selector/item-selector-web/src/main/java/com/liferay/item/selector/web/internal/portlet/ItemSelectorPortlet.java

```diff
@@ -69,7 +74,25 @@ public void render(
 
 		localizedItemSelectorRendering.store(renderRequest);
 
-		super.render(renderRequest, renderResponse);
+		try {
+			PermissionChecker userPermissionChecker =
```

Comment:
@bakayattila, I would just rename to permissionChecker

Reply:
will change it after ci:relevant. I wanted to test it first to see if I break any tests with this change.

Reply:
@jkappler done